src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,23 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    private Random random = new Random();
+
+    @Override
+    public List<? extends Machine> get() {
```

Comment:
In producer realization, you should have the exact type of list, not the wildcard. Fix it in all producers